docs/projects/ansible/collection/centurion/playbooks/inventory.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Inventory
+description: Inventory host machines and publish to Centurion ERP
+date: 2024-08-19
+template: project.html
+about: https://github.com/nofusscomputing/ansible_collection_centurion
+---
+
+The inventory playbook has been created to inventory host machines and to publish the collected inventory to Centurion ERP. The inventory includes details of all software packages installed on the host machine as well as some details regarding the host machine such as UUID and serial number.
```

Comment:
```suggestion
The inventory playbook has been created to inventory devices and to publish the collected inventory to Centurion ERP. The inventory includes details of all software packages installed on the host machine as well as some details regarding the host machine such as UUID and serial number.
```

stick to the interface terms as this aids in clarity

Reply:
Updated